packages/shared/lib/contexts/onboarding/actions/completeOnboardingProcess.ts

```diff
@@ -0,0 +1,20 @@
+import { get } from 'svelte/store'
+
+import { createNewAccount } from '@core/account'
+import { login } from '@core/profile'
+
+import { ProfileSetupType } from '../enums'
+import { onboardingProfile } from '../stores'
+
+import { addOnboardingProfile } from './addOnboardingProfile'
+import { cleanupOnboarding } from './cleanupOnboarding'
+import { initialiseSecretManager } from './initialiseSecretManager'
+
+export async function completeOnboardingProcess(): Promise<void> {
+    addOnboardingProfile()
+    await initialiseSecretManager()
```

Comment:
Do we need this in the onboarding? Shouldnt this have already happened?

Reply:
We don't - the underlying `restoreFromBackupFile` is already being called in the actual flow. I think the changes that Thoralf is making will fix this, b/c it seems to me it's still looking at the wrong path somehow.